core/src/main/scala/com/raphtory/package.scala

```diff
@@ -0,0 +1,34 @@
+package com
+
+import cats.effect.Resource
+import cats.effect.Sync
+import com.oblac.nomen.Nomen
+import com.raphtory.internals.management.GraphConfig.ConfigBuilder
+import com.raphtory.internals.management.id.LocalIDManager
+import com.raphtory.internals.management.id.ZooKeeperCounter
+import com.raphtory.internals.management.id.ZookeeperLimitedPool
+import com.typesafe.config.Config
+
+package object raphtory {
+
+  private[raphtory] def createName: String = Nomen.est().adjective().color().animal().get()
+
+  private[raphtory] val defaultConf: Config = ConfigBuilder().build().getConfig
```

Comment:
We could use this method in all the places we have `ConfigBuilder().build().getConfig`

Reply:
- Internally, yes! But there seems no way to constraints this. 
- From client projects `ConfigBuilder` is available to users to be able create fresh config instance.